WordPress/src/main/java/org/wordpress/android/util/WPMediaUtils.java

```diff
@@ -84,6 +85,9 @@ public static Uri getOptimizedMedia(Context context, String path, boolean isVide
             AppLog.e(AppLog.T.EDITOR, "Optimized picture was null!");
             AnalyticsTracker.track(AnalyticsTracker.Stat.MEDIA_PHOTO_OPTIMIZE_ERROR);
         } else {
+            // Set the default orientation tag for the EXIF data
+            exifData.put("Orientation", String.valueOf(ExifInterface.ORIENTATION_NORMAL));
```

Comment:
The image I was testing with didn’t have an orientation value set in its EXIF data, but it was being assigned the `Orientation=6` value, which corresponds to [ORIENTATION_ROTATE_90](https://developer.android.com/reference/android/media/ExifInterface#ORIENTATION_ROTATE_90).

Reply:
I couldn't find any documentation about what `ORIENTATION_NORMAL` does - but I assume it says to keep the original orientation 🤷‍♂️